ghproxy/ghcache/coalesce.go

```diff
@@ -50,15 +50,18 @@ type responseWaiter struct {
 // Notes: Deadlock shouldn't be possible because the map lock is always
 // acquired before responseWaiter lock if both locks are to be held and we
 // never hold multiple responseWaiter locks.
-func (r *requestCoalescer) RoundTrip(req *http.Request) (*http.Response, error) {
+func (r *requestCoalescer) RoundTrip(req *http.Request) (resp *http.Response, err error) {
```

Comment:
This function is more than a few lines long, plesae remove the named result params

https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters